.github/workflows/promci.yml

```
@@ -0,0 +1,31 @@
+name: dagger
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
```

Are we still doing windows tests?

```suggestion
    strategy:
      matrix:
        os: [ubuntu-latest, windows-latest]
    runs-on: ${{ matrix.os }}
```